New feature

aron-net opened the pull request.

aron-net committed:
Lay out of the page without implement any functionality.

aron-net committed:
fix linters

aron-net committed:
Fixing the stylelint

aron-net committed:
Trying to error stylelint

aron-net committed:
Trying to error stylelint

aron-net committed:
Delete file--home-milu-Desktop-mobile-version-mobile_viersion.html

aron-net committed:
Installing hintrc and stylintrc to fix the error of linter.

aron-net committed:
Merge remote-tracking branch 'refs/remotes/origin/feature' into feature

aron-net committed:
fixing index html and css

Mre55 commented on index.html:
```diff
@@ -0,0 +1,86 @@
+</head>
+<body>
+    <header class="top-navigation">
+        <a href="#"><img src="./Image/John_Doe.png" class="log" alt=" My Logo"></a>
```
**[OPTIONAL]** As this is your portfolio and the logo should have to be your name and please remove the image and replace it with the text of your name.

Mre55 reviewed (changes requested):
**Hi @aron-helu,**

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Highlights 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The commit messages are meaningful ✅ 
- Linters are working properly ✅ 

 ## Required Changes ♻️

- [ ] Kindly, add pull request description to your pull request currently it's empty since you create new branch for this changes. For your future reference you have to continue on the existing branch to make your changes you don't have to create new branch for each comments and changes.

<img width="935" alt="image" src="https://user-images.githubusercontent.com/34578631/170783420-7c25960d-6699-4e7f-870a-66a3f7b3130e.png">


_Check the comments under the review._

<img width="430" src="https://media1.giphy.com/media/Jq824R93JsLwZCaiSL/giphy.gif"/>


## Optional suggestions 🟡 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

aron-net committed:
Update index.html

aron-net committed:
Log style

Update style for the log.

aron-net committed:
Update style

vechicin commented on style.css:
```diff
@@ -0,0 +1,216 @@
+  font-weight: bolder;
+  text-align: left;
+  color: #9390ff;
+}
```
Your logo doesn't quite look like the one from the template design.

Here's your logo:

<img width="93" alt="Screen Shot 2022-05-27 at 15 56 42" src="https://user-images.githubusercontent.com/82426682/170788493-eb94a4ab-e83a-4798-b5a8-88b8a9b80953.png">

Here's the template design logo:

<img width="71" alt="Screen Shot 2022-05-27 at 15 57 03" src="https://user-images.githubusercontent.com/82426682/170788523-026e9bae-f63e-4f43-8232-a34eac120f5d.png">

Kindly fix it.

vechicin commented on style.css:
```diff
@@ -0,0 +1,216 @@
+}
+
+/* Item navigation */
+.item-navigation {
+  display: flex;
+  flex-direction: row;
+  flex: 2;
+  gap: 2px;
+}
```
Your hamburger button doesn't quite look like the one from the template design.

Here's your hamburger:

<img width="39" alt="Screen Shot 2022-05-27 at 15 58 41" src="https://user-images.githubusercontent.com/82426682/170788700-81cf4a47-1c92-4862-8f59-5fc946a5a8d2.png">

Here's the template design hamburger:
<img width="50" alt="Screen Shot 2022-05-27 at 15 59 02" src="https://user-images.githubusercontent.com/82426682/170788744-3e63d24f-a508-4f48-8a8d-6a3a9da7bf56.png">

Do you see how different they are in size? Kindly fix this.

vechicin commented on index.html:
```diff
@@ -0,0 +1,86 @@
+            </p>
+        </div>        
+    </section>
+    <div class="thematic">
```
The content rendered starting at this line until the end of the file should be moved inside the `<section>` container.

I would also recommend you don't use the `<main>` tag unless the content you render inside of it actually is the page's main content. Please change all of your `<main>` tags to `<div>` tags to comply with HTML best practices.

vechicin commented on style.css:
```diff
@@ -0,0 +1,216 @@
+}
+
+/* Section Header */
+.header-text {
+  font-family: 'Roboto', sans-serif;
+  font-weight: bold;
+  font-size: 40px;
+  line-height: 72px;
+  color: #9391ff;
+  margin-top: 2%;
+  text-align: center;
+}
+
+/* Support text of header */
+.support-text {
+  font-family: 'Roboto', sans-serif;
+  font-weight: lighter;
+  font-size: 32px;
+  line-height: 64px;
+  text-align: center;
+}
```
Both of your titles don't quite look like the ones from the template design.

Here are your titles:

<img width="192" alt="Screen Shot 2022-05-27 at 16 08 05" src="https://user-images.githubusercontent.com/82426682/170789638-2f8ac2d8-9a40-4ba0-a9a2-70a1d868cdc7.png">

Here are the template design titles:

<img width="226" alt="Screen Shot 2022-05-27 at 16 08 33" src="https://user-images.githubusercontent.com/82426682/170789684-31500d70-f0bd-43d2-97a6-acb8a3101ba1.png">

Do you see how different they are in size and how they are styled in normal font as opposed to _italic_? Kindly fix this as following the template as close as possible is a project requirement.

vechicin commented on style.css:
```diff
@@ -0,0 +1,216 @@
+}
+
+/* Language list container */
+.language-container {
+  display: flex;
+  flex-direction: column;
+  width: 25%;
+  height: 25vh;
+}
+
+/* Frame-work list container */
+.frame-works-container {
+  display: flex;
+  flex-direction: column;
+  width: 25%;
+  height: 20vh;
+  text-align: left;
+}
+
+/* Skill list container */
+.skills-container {
+  display: flex;
+  flex-direction: column;
+  width: 28%;
+  height: 20vh;
+  text-align: left;
+  margin-bottom: 10%;
+}
```
This section doesn't quite match the template design.

Here's your design:

<img width="243" alt="Screen Shot 2022-05-27 at 16 11 17" src="https://user-images.githubusercontent.com/82426682/170789971-f76084f1-b2ec-46fe-96f9-128a77c01d95.png">

Here's the template design:

<img width="289" alt="Screen Shot 2022-05-27 at 16 11 35" src="https://user-images.githubusercontent.com/82426682/170790012-da45fc03-4d90-4d81-a62f-77eecb284e0f.png">

You should fix the `font-family`, the `font-size`, the `margins` between each technology and the one below it, and the `margins` between the containers and the edge of the screen. The dividers are too thick as well. Kindly fix this.

vechicin commented on style.css:
```diff
@@ -0,0 +1,216 @@
+}
+
+/* Social media links container */
+.account-links {
+  display: flex;
+  width: 90%;
+  justify-content: center;
+  margin-top: 5%;
+}
```
Your social media icons don't quite match the ones from the template design.

Here's your design:

<img width="249" alt="Screen Shot 2022-05-27 at 16 15 50" src="https://user-images.githubusercontent.com/82426682/170790403-50ef9030-810b-4931-9938-6f4332a5b989.png">

Here's the template design:

<img width="285" alt="Screen Shot 2022-05-27 at 16 16 14" src="https://user-images.githubusercontent.com/82426682/170790446-0f9fa60a-d834-4897-8f14-ff386f586dbb.png">

I suggest you export the icons directly from Figma. You should also fix the margins between the icons and make sure they are the right size. Kindly fix this.

vechicin commented on style.css:
```diff
@@ -0,0 +1,216 @@
+}
+
+/* work button */
+button {
+  display: flex;
+  flex-direction: row;
+  width: 30%;
+  height: 5vh;
+  margin-left: 25%;
+  margin-top: 8%;
+  border: none;
+  font-weight: bold;
+  font-size: 22px;
+  line-height: 32px;
+  color: #a2a3ae;
+}
```
Your `Work` button doesn't quite match the one from the template design.

Here's your button:

<img width="249" alt="Screen Shot 2022-05-27 at 16 18 14" src="https://user-images.githubusercontent.com/82426682/170790633-7837ad56-129a-4c11-8803-45ed5b7a8bea.png">

Here's the template design button:

<img width="284" alt="Screen Shot 2022-05-27 at 16 18 40" src="https://user-images.githubusercontent.com/82426682/170790665-daeb85db-0ca2-45e1-9da1-a42e3d885a0c.png">

It needs to be positioned in the middle of the page, it shouldn't have a border and the ⬇️ doesn't quite look the same. Kindly fix this.

vechicin reviewed (changes requested):
# Hi @aron-helu  ✋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet. 🚨 


 ## Highlights ⚡ 

- [x] Good commit history.

 ## Required Changes ♻️

- [ ] The previous reviewer asked you to provide a PR description and you didn't. Additionally, your PR title needs to be changed because it is not descriptive of what a reviewer would find in the branch at all. You need to be really careful with your PR title/description because it is reason to invalidate your review request and not having a proper title/description could prevent you from moving forward in the program, which is obviously not ideal.

_Check the comments under the review._

## Optional suggestions 📂 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

aron-net committed:
Update style.css

aron-net committed:
Fixing log

Fixing the log color for the mobile version. Layout elements of the page without implementation and functionality.

aron-net committed:
Failer stylelint

Failer of color style for log and text.
Fixing by changing the color-hex typing error.

aron-net committed:
Error linter stylelint 

Failer of color style for log and text.
Fixing by changing the color-hex typing error.
